pkg/config/testdata/containers_default.conf

```diff
@@ -287,7 +287,7 @@ runtime_supports_json = ["runc"]
 # SSH config file path
 ssh_config = "/foo/bar/.ssh/config"
 
-# Number of seconds to wait for podmansh logins.
+# Deprecated in favor of podmansh.Timeout, should not be used anymore.
```

Comment:
Nit: Maybe we could add a note here that it'd be removed in Podman 6. @rhatdan I'm hoping this won't be trouble to remove for podman 6.

Reply:
ah it's just testdata which I don't think matters much to end users.

Reply:
Yes, it is removed from the actual `containers.conf`. I mark this as resolved, feel free to reopen if there's something more.